LoopKitUI/Extensions/Environment+Colors.swift

```diff
@@ -109,6 +109,7 @@ private struct ChartColorPaletteKey: EnvironmentKey {
     static let defaultValue: ChartColorPalette = ChartColorPalette(axisLine: .clear,
                                                                    axisLabel: .secondaryLabel,
                                                                    grid: .systemGray3,
+                                                                   presetTint: .systemIndigo,
```

Comment:
@ps2 I just picked the closest system color I could find that wasn't being used here. Happy to adjust this default if you'd like -- not too sure where this fallback value would even be shown.